docs/sources/flow/reference/components/loki.source.file.md

```diff
@@ -25,51 +25,57 @@ loki.source.file "LABEL" {
 ```
 
 ## Arguments
+
 The component starts a new reader for each of the given `targets` and fans out
 log entries to the list of receivers passed in `forward_to`.
 
 `loki.source.file` supports the following arguments:
 
- Name         | Type                 | Description                                                | Default | Required 
---------------|----------------------|------------------------------------------------------------|---------|----------
- `targets`    | `list(map(string))`  | List of files to read from.                                |         | yes      
- `forward_to` | `list(LogsReceiver)` | List of receivers to send log entries to.                  |         | yes      
- `encoding`   | `string`             | The encoding to convert from when reading files.           | `""`    | no       
+| Name             | Type                 | Description                                                                       | Default | Required |
+| ---------------- | -------------------- | --------------------------------------------------------------------------------- | ------- | -------- |
+| `targets`        | `list(map(string))`  | List of files to read from.                                                       |         | yes      |
+| `forward_to`     | `list(LogsReceiver)` | List of receivers to send log entries to.                                         |         | yes      |
+| `encoding`       | `string`             | The encoding to convert from when reading files.                                  | `""`    | no       |
+| `start_from_eof` | `bool`               | Whether the file should be tailed from the end if a cached position is not found. | `false` | no       |
 
 The `encoding` argument must be a valid [IANA encoding][] name. If not set, it
 defaults to UTF-8.
 
+The `start_from_eof` argument is valuable when you want to tail a large file without reading its entire content.
+When set to true, only new logs will be read, ignoring the existing ones.
+
```

Comment:
I formatted this document in a specific commit, I hope this does not make the review too annoying. I find the doc easier to read/modify when it is nicely formatted,